assembl/static2/js/app/components/common/visibilityComponent.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
I don't understand the meaning of `visibilityComponent`. Can you explain? May be we can find a better name for this react component...